include/PortableBuffers.hpp

```diff
@@ -1,22 +1,28 @@
 #pragma once
 
 #ifndef SAPF_AUDIOTOOLBOX
-#include <cstdint>
```

Comment:
isnt this needed for `uint32_t` and friends?

Reply:
my IDE told me it was "possibly unused" and it does seem to work without it, but I assume that's because it's indirectly getting included via VM.hpp, which I assume it's not a good practice to rely on. Will add it back.